scripts/generate.all.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -xeo pipefail
+
+find ./integration-tests -mindepth 1 -maxdepth 1 -type d | xargs -n1 -P 8 basename | xargs -n1 -P 8 ./scripts/generate.template.sh
```

Comment:
There seems to be some kind of race condition where sometimes silent failures occur :disappointed: - but generally it seems to be working well and is now 5s wall clock time on my machine vs ~30s